src/nvim/menu.c

```diff
@@ -1156,13 +1154,17 @@ static char_u *popup_mode_name(char_u *name, int idx)
 }
 
 
-/*
- * Duplicate the menu item text and then process to see if a mnemonic key
- * and/or accelerator text has been identified.
- * Returns a pointer to allocated memory, or NULL for failure.
- * If mnemonic != NULL, *mnemonic is set to the character after the first '&'.
- * If actext != NULL, *actext is set to the text after the first TAB.
- */
+/// Duplicate the menu item text and then process to see if a mnemonic key
+/// and/or accelerator text has been identified.
+///
+/// If mnemonic != NULL, *mnemonic is set to the character after the first '&'.
+/// If actext != NULL, *actext is set to the text after the first TAB.
+///
+/// @param str
+/// @param mnemonic
+/// @param actext
+///
+/// @return a pointer to allocated memory.
 static char_u *menu_text(char_u *str, int *mnemonic, char_u **actext)
```

Comment:
We can add attributes `FUNC_ATTR_NONNULL_RET` and `FUNC_ATTR_WARN_UNUSED_RESULT`.